source/app/pages/home.js

```diff
@@ -58,7 +58,7 @@ function Home({ dispatch, tracks }) {
       </div>
 
       {/* <p>
-        <img src="/static/images/icon_feature.svg" /> If you see one of these along the way, click
+        <img src="/images/icon_feature.svg" /> If you see one of these along the way, click
```

Comment:
If this is a comment and not needed, I think we can remove it

Reply:
Agreed we should not have commented out code unless it is for documentation. I just did a global find/replace so didn't notice this.